test/e2e/certs.go

```diff
@@ -32,6 +32,9 @@ var _ = Describe("MasterCerts", func() {
 	})
 
 	It("should have all expected certs on the master", func() {
+		if testContext.Provider == "" {
```

Comment:
I would have much preferred a shared utility function along the lines of:

ProviderEquals(string) bool

... than cut-n-pasting this logic all over the place like this.  But lets get this merged now, and fix the aforementioned in a separate PR.  Please open an issue for that.